lib/utils.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+let nodeutil = null;
+try {
+  nodeutil = require('util');
+} catch (e) {
+  // ignore
+}
+
+const inspect = nodeutil
+  ? val => nodeutil.inspect(val, { depth: null, sorted: true })
+  : val => JSON.stringify(val, null, 2);
+
+const format = nodeutil
+  ? (...args) => nodeutil.format(...args)
```

Comment:
Something like `format('%s:%s', 'abc', 'qwe')` will produce different outputs.

Reply:
Yes, that's as expected.

Reply:
I don't like the fact that if there is a function that takes, for example, takes arguments like those mentioned above reporter will output incorrect values.